tests/testthat/test-arg.R

```diff
@@ -17,6 +17,14 @@ test_that("informative error message on partial match", {
   )
 })
 
+test_that("informative error message on a typo", {
+  myarg <- "continuuos"
+  expect_error(
+    arg_match(myarg, c("discrete", "continuous")),
```

Comment:
I think you could inline `myarg` here?

Reply:
For posterity, inlining returns `Error: Internal error: `arg_match()` expects a symbol`